tests/phpunit/elementor/includes/template-library/test-manager-remote.php

```diff
@@ -4,29 +4,48 @@
 use ElementorEditorTesting\Elementor_Test_Base;
 
 class Elementor_Test_Manager_Remote extends Elementor_Test_Base {
+	const ELEMENTOR_LIBRARY_REMOTE_META_KEY = 'elementor_library_remote';
+	const FAVORITES = 'favorites';
 
 	/**
 	 * @var \Elementor\TemplateLibrary\Manager
 	 */
 	protected static $manager;
 	private $fake_template_id = '777';
+	private $second_fake_template_id = '123';
+
 
 
 	public static function setUpBeforeClass() {
 		self::$manager = self::elementor()->templates_manager;
 	}
 
 	public function test_should_mark_template_as_favorite() {
-		$this->assertFalse(
-			self::$manager->mark_template_as_favorite(
-				[
-					'source' => 'remote',
-					'template_id' => $this->fake_template_id,
-					'favorite' => 'false',
-				]
-			)
-		);
+        $user= $this->act_as_admin();
```

Comment:
Please follow the CS in whole file
```suggestion
        $user = $this->act_as_admin();
```

Reply:
Done